src/pages/Account/Tabs/ModulesTab/Tabs.tsx

```diff
@@ -39,13 +39,13 @@ function TabPanel({value, address}: TabPanelProps): JSX.Element {
 function ModulesTabs({address}: {address: string}): JSX.Element {
```

Comment:
switch btw "read" and "write" tab doesn't maintain the search param.

Reply:
That's currently expected since search param has currently no function in the write tab (deep link support for the write tab is not implemented) and we currently don't link to the write tab from anywhere anyways.
It's certainly something we can consider to implement before graduating the "write" (or rather Run) functionality from dev mode to GA, but I was under the impression that the Run mode needs some further changes anyways so I didn't want to touch it too much (but also not break existing functionality).